dynawo/sources/Common/Dictionaries/DYNTimeline_en_GB.dic

```diff
@@ -40,6 +40,10 @@ PhaseShifterBelowMin      =             Phase-shifter : below minimum allowed va
 PhaseShifterBelowMax      =             Phase-shifter : below maximum allowed value
 PhaseShifterBelowStop     =             Phase-shifter : back to acceptable value
 PhaseShifterWithinInterval=             Phase-shifter : back to acceptable value
+// --> Phase-shifter Lock
+PhaseShifterBlockingIActing =           Phase-shifter : order to lock
+PhaseShifterBlockingIArming =           Phase-shifter : lock arming by over-current constraint
+PhaseShifterBlockingIDisarming =        Phase-shifter : lock disarming by under-current constraint
```

Comment:
Not exactly and under-current I would say but only that we are now not in over-current any more no ?

Reply:
comment updated